### PR TITLE
version 1.1.0

### PR DESCRIPTION
**Pull request by nyuichi**
After this pr is merged I will add tag 1.1.0 and publish it to na4zagin3/satyrographos-repo.

# Changelog since 1.0.0

- #19 Refactor Derive by zeptometer
- #18 add standalone-like class file by nyuichi
- #15 Visual regression test by zeptometer
- #17 add: Context module by nyuichi
- #16 fix(pager): fix type signatures by nyuichi
- #11 color型に関する比較用関数の追加 by puripuri2100

### CODE REVIEW

**Comment by nyuichi** [comment]
@na4zagin3 

Would you mind reviewing this PR?

**Comment by nyuichi** [comment]
Good catch! BTW I had the same bug in another pr so we should introduce some mechanism to check this kind of bug.

**Comment by nyuichi** [comment]
Thank you! 🙏

**Comment by na4zagin3** [comment]
I'll address the issue with https://github.com/na4zagin3/satyrographos/issues/64.

I'm planning to add a declaration `(packageDir <dir>)` that specifies a directory having package files instead of `(package ...)` which specifies a single file. How do you think?

**Comment by nyuichi** [comment]
Amazing! That should work for this repository. I guess there might be some alternatives to the name of the declarations, perhaps like `package-dir` and `directory`, while I have no strong opinion on this. And other satysfi users may want to use declarations like `(packageDir <src> <dest>)`. I am not sure how this is really useful and needed though.

I am going to gather all files and directories containing actual satyh/satyg into a single directory called `src`. So for our own purpose simply adding a directive like `(packageDir "src")` will suffice.

File: Satyristes [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
   (name "base")
-  (version "1.0.0")
+  (version "1.1.0")
   (sources
     ((package "array.satyg" "array.satyg")
      (package "base.satyg" "base.satyg")
      (package "base0.satyg" "base0.satyg")
      (package "block.satyh" "block.satyh")
      (package "char.satyg" "char.satyg")
      (package "color-ext.satyg" "color-ex.satyg")
```
Comment (na4zagin3) [inline Satyristes]: Oops, I overlooked `"color-ex.satyg"`, which must be `"color-ext.satyg"`.

**Pull request merged**